src/main/java/dwbh/api/domain/input/UserGroupAndVisibleMemberListInput.java

```diff
@@ -24,7 +24,7 @@
  *
  * @since 0.1.0
  */
-public class UserGroupInput {
+public class UserGroupAndVisibleMemberListInput {
```

Comment:
I think the name of the inputs should be tied to the name of the operation they are going to be used as parameters. In this case this input is going to be used for the `listUsersGroup` operation therefore this should be `ListUsersGroupInput`. Is shorter and tells me for what operation is going to be used.

Reply:
Agree. Done